core/src/wallet/ethereum/explorers/LedgerApiEthereumLikeBlockchainExplorer.cpp

```diff
@@ -275,13 +275,18 @@ namespace ledger {
                     });
         }
 
-        Future<String> LedgerApiEthereumLikeBlockchainExplorer::pushLedgerApiTransaction(const std::vector<uint8_t> &transaction) {
+        Future<String> LedgerApiEthereumLikeBlockchainExplorer::pushLedgerApiTransaction(const std::vector<uint8_t> &transaction, const std::string& correlationId) {
             std::stringstream body;
             auto hexTx = "0x" + hex::toString(transaction);
             body << "{" << "\"tx\":" << '"' << hexTx << '"' << "}";
             auto bodyString = body.str();
+            std::unordered_map<std::string, std::string> headers;
+            if(!correlationId.empty()) {
+                headers["correlationId"] = correlationId;
```

Comment:
The header should be named `ledger.correlation_id` according to https://ledgerhq.atlassian.net/wiki/spaces/VAUL/pages/2886074741/Logging+Guidelines#Normalising-the-data